package.json

```diff
@@ -33,7 +33,7 @@
     "cloudflare-middleware": "^1.0.4",
     "config": "^3.3.6",
     "cross-env": "^7.0.3",
-    "decamelize": "^6.0.0",
+    "decamelize": "^5.0.1",
```

Comment:
You should pin it at [version 3](https://github.com/sindresorhus/decamelize/releases/tag/v4.0.0). The `humanize-string` needs to upgraded as well.

```diff
# dependencies
-    "decamelize": "^6.0.0",
+    "decamelinze": "3.2.0",

# devDependencies
- "humanize-string": "^3.0.0",
+ "humanize-string": "2.0.0",
```